src/components/Flex/__snapshots__/flex.test.js.snap

```diff
@@ -10,7 +10,7 @@ exports[`Flex component matches snapshot (default props) 1`] = `
 
 exports[`Flex component matches snapshot (set all props) 1`] = `
 <div
-  className="flex flex--alignEnd flex--column flex--column--reverse flex--justifySpaceBetween flex--noGutters"
+  className="flex--column--reverse flex--justifySpaceBetween flex--noGutters flex flex--alignEnd flex--column"
```

Comment:
This is when I like snapshots - we don't need to test classcat, as it's already covered by unit tests. We only need to see that usage has the desired output.

Reply:
hmmmmmmmmm 🙃